starknet_py/tests/e2e/account/outside_execution_test.py

```diff
@@ -5,43 +5,40 @@
 from starknet_py.constants import ANY_CALLER, OutsideExecutionInterfaceID
 from starknet_py.hash.selector import get_selector_from_name
 from starknet_py.net.account.account import BaseAccount
-from starknet_py.net.client_models import Call, OutsideExecutionTimeBounds
-from starknet_py.tests.e2e.fixtures.constants import MAX_FEE
+from starknet_py.net.client_models import (
+    Call,
+    OutsideExecutionTimeBounds,
+    ResourceBounds,
+)
 from starknet_py.transaction_errors import TransactionRevertedError
 
 
 @pytest.mark.asyncio
 async def test_argent_account_outside_execution_compatibility(
     argent_account: BaseAccount,
+    argent_account_v040: BaseAccount,
 ):
-    result = await argent_account.supports_interface(OutsideExecutionInterfaceID.V1)
-    assert result is True
-    result = await argent_account.supports_interface(OutsideExecutionInterfaceID.V2)
-    assert result is False
+    for a, has_v1, has_v2 in [
+        (argent_account, True, False),
+        (argent_account_v040, True, True),
+    ]:
+        assert await a.supports_interface(OutsideExecutionInterfaceID.V1) is has_v1
+        assert await a.supports_interface(OutsideExecutionInterfaceID.V2) is has_v2
```

Comment:
nit: Full name seems better than a single letter 😅 
```suggestion
    for account, has_v1, has_v2 in [
        (argent_account, True, False),
        (argent_account_v040, True, True),
    ]:
        assert await account.supports_interface(OutsideExecutionInterfaceID.V1) is has_v1
        assert await account.supports_interface(OutsideExecutionInterfaceID.V2) is has_v2
```